custom_components/tuya_local/devices/gosund_SW2_dimmer_switch.yaml

```diff
@@ -0,0 +1,28 @@
+name: Gosund SW2 Dimmer Switch
```

Comment:
This name should be generic. With such a simple device, there is high likelihood that the config will match other brands.

Reply:
Got it - what's a good generic choice? `Dimmer Switch`?

Reply:
Yes, but following HA naming convention, "Dimmer switch"